elasticsearch/build.gradle.kts

```diff
@@ -40,7 +46,7 @@ dependencies {
     // https://github.com/elastic/elasticsearch/issues/49218
```

Comment:
Is this issue not fixed in elastic 7.15.2? It looks like it was fixed a couple years ago...

Reply:
Yes, this issue is already solved, and I've removed the constraints. Thanks!